### ISSUE TITLE
Question about out_scale=0.001

### ISSUE THREAD

**Issue by zzf005** [huanglianghua/siamfc-pytorch#24]
Hi @huanglianghua 
I have a question about the hyperparameter out_scale in heads.py
Why this parameter is necessary? I can't find this parameter in the official code (Matlab ver) and in the paper.
Hope to find some inspirations.
Anyway, Thank you for your great work.

**Comment by laisimiao** [comment]
me too @huanglianghua

**Comment by huanglianghua** [comment]
The `out_scale` is used for scaling down the logits to around 0~10, so that the BCEWithLogits loss can be easier to optimize.

The original score maps generated by `z * x` is way too large (around like 1000), and a sigmoid function will push the values far away from 0.5, where gradients are small. An `out_scale` avoids this.

As I remember, the official version of SiamFC also uses the `out_scale`, since this implementation was adapted from the MATLAB version.

**Comment by zzf005** [comment]
@huanglianghua got it. Thanks!
